_layouts/base.html

```diff
@@ -22,7 +22,7 @@
     {% endif %}
     {% seo title=false %}
     {% feed_meta %}
-    <meta name="viewport" content="width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Why is this needed? Can this be scoped just to the list?

Reply:
This is not needed to fix the bug itself, but to prevent other potential overflows from causing the page to "scale down", like in the first screenshot. Sadly it cannot be scoped.

That said, any overflow needs to be addressed anyway, so it does not matter a lot 😅

Reply:
@kidonng If it's not needed to fix this bug: can you remove it? Thanks!

Reply:
Oh, you did, sorry!